src/site/site.xml

```diff
@@ -46,6 +46,7 @@
       <!-- Will be added to every generated page -->
       <![CDATA[
         <script type="text/javascript" src="$relativePath/js/checkstyle.js"></script>
+        <script type="text/javascript" src="$relativePath/js/redirect.js"></script>
```

Comment:
For the config_xxxx.xml approach, we will still require this extra JS file as we cannot write JS inside `<script>` tags in xml files. And we are importing this JS file in all the generated HTML files, I'm not sure how we can import this JS file into only required html files, i.e config_coding.html, config_header.html, etc.